code/game/objects/structures/flaps.dm

```diff
@@ -9,6 +9,27 @@
 	pass_flags_self = PASSGLASS
 	var/airtight = 0
 
+/obj/effect/plasticflaps_overlay
+	icon = 'icons/obj/stationobjs.dmi' //Change this.
```

Comment:
//Change this.
???

Reply:
i don't really recall why i added that comment, or if it came from something else via copy pasting, i'll fix it